airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/BigQueryDestination.kt

```diff
@@ -0,0 +1,394 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+package io.airbyte.integrations.destination.bigquery
+
+import com.codepoetics.protonpack.Indexed
+import com.codepoetics.protonpack.StreamUtils
```

Comment:
do we want a dependency on some obscure package that hasn't been maintained in 3 years?

Reply:
tried removing but it has some zip the stream method which we have to implement or use alternative (not difficult just ignored in this PR). Will take a pass later in the refreshes PR.